distributed-job-manager/src/main/java/ru/fix/distributed/job/manager/Manager.java

```diff
@@ -86,6 +99,49 @@ public void start() throws Exception {
 
         leaderLatch.start();
         workersAliveChildrenCache.start();
+        startWorkPoolCleaningTask();
+    }
+
+    private void startWorkPoolCleaningTask() {
+            workPoolCleaningReschedulableScheduler.schedule(
```

Comment:
start delay is Zero, better to postpone cleaning process and run at after workPoolCleanPeriodMs delay

Reply:
fixed

Reply:
it should be noted that startDelay is a constant in ReschedulableScheduler. So, if someone will type by mistake a couple extra zeros to this property, he won't be able fix it without restarting DJM